engine/src/main/java/org/hibernate/search/query/engine/impl/LuceneHSQuery.java

```diff
@@ -66,7 +65,7 @@
  * @author Emmanuel Bernard
  * @author Hardy Ferentschik
  */
-public class LuceneHSQuery extends AbstractHSQuery implements HSQuery, Serializable {
```

Comment:
> In practice, it's LuceneHSQuery which you need to be serializable, right?
 
Yep, that's the one causing trouble, although I haven't tested yet with ```ElasticSearchHSQueryImpl``` so I though it'd make sense for both to be Serializable